config/systemd/omnect-ui.service

```diff
@@ -0,0 +1,42 @@
+[Unit]
+Description=omnect ui
+After=time-sync.target docker.service omnect-device-service.service
```

Comment:
is Wants+After docker.service sufficient? means: does docker notify systemd when it is successfully started?
if not i would wait with the start until the docker socket file exist.

Reply:
I assume docker is fully initialized before
```
[root@jza-gateway-devel-rpi omnect]# cat /lib/systemd/system/docker.service
[Unit]
...

[Service]
Type=notify

```